src/rtcModule/webrtc.cpp

```diff
@@ -2621,7 +2621,21 @@ void Call::onClientLeftCall(Id userid, uint32_t clientid)
         }
 
         SUB_LOG_DEBUG("ENDCALL received for ourselves, finishing the call");
-        destroy(TermCode::kErrNetSignalling, false);
+        auto wptr = weakHandle();
+        destroy(TermCode::kErrNetSignalling, false)
+        .then([wptr, this]
```

Comment:
Why do we want a `wptr` if it's not checked in the lambda?? :stuck_out_tongue:

Reply:
Fixed